packages/kit/Cargo.toml

```diff
@@ -9,4 +9,4 @@ repository = "https://github.com/bandprotocol/owasm/tree/master/packages/kit"
 version = "0.2.2"
```

Comment:
should we update the version of the package as well? as it breaks the old function.